src/components/table/TableCellRenderer.tsx

```diff
@@ -15,6 +15,9 @@ const useStyles = makeStyles((theme: Theme) => ({
   date: {
     whiteSpace: 'nowrap',
   },
+  default: {
+    borderBottom: `1px solid ${theme.palette.TwClrBrdrSecondary} !important`
```

Comment:
`!important` is usually frowned upon, guessing it's ok here since we don't expect to override this style.